### PR TITLE
Added lightz96 portfolio

### PR DESCRIPTION
**Pull request by lightz96**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -730,92 +730,114 @@ image::clone/cloneDG/CloneActivityDiagram.png[]
...
+
+// tag::designconsideration[]
 The following activity diagram shows what happens if the user executes a command:
 
 .Activity diagram of the command execution.
 image::UndoAndRedo/RunCommandActivityDiagrams.png[]
 
 ==== Design considerations
 
+This section shows some of the design considerations taken before implementing the undo and redo features.
```
Comment (krusagiz) [inline docs/DeveloperGuide.adoc]: It should be:
... considerations taken _when_ implementing ...

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -730,92 +730,114 @@ image::clone/cloneDG/CloneActivityDiagram.png[]
...
+
+===== Aspect: Algorithm used to implements undo and redo features
+
+* Alternative 1 (current choice): Provides different sets of implementation for each command so that the command knows how to undo and redo itself.
```
Comment (krusagiz) [inline docs/DeveloperGuide.adoc]: How about:
_Provide_ different _implementations_ for each command ...

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -730,92 +730,114 @@ image::clone/cloneDG/CloneActivityDiagram.png[]
...
+
 ===== Aspect: Data structure used to keep track of undoable command
 
-* **Alternative 1 (current choice):** Use two stacks. One stack stores the command to undo while the other stack stores the command to redo.
-** Pros: Easy to implement.
-** Cons: Need to carefully manage the two stacks. For example, you need to pop the command from one stack and push it into another stack.
-* **Alternative 2:** Use an arraylist and a pointer to indicate the next command to undo.
-** Pros: Easy to implement.
-** Cons: Need to carefully manage the pointer. For example, you need to ensure that the pointer stays within the size of an arraylist.
+* Alternative 1 (current choice): Uses two stacks. One stack stores the command to undo while the other stack stores the command to redo.
```
Comment (krusagiz) [inline docs/DeveloperGuide.adoc]: It should be:
_Use_ two stacks.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -730,92 +730,114 @@ image::clone/cloneDG/CloneActivityDiagram.png[]
...
+** Pros: Makes the implementation easy to code.
+** Cons:  Needs to manage the two stacks carefully. For example, you need to pop the command from one stack and push it into another stack.
+* Alternative 2: Uses a list and a pointer to indicate the next command to undo.
```
Comment (krusagiz) [inline docs/DeveloperGuide.adoc]: It should be:
_Use_ a list ...

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -483,13 +475,15 @@ To `undo` the `delete` command:
 
 . Type *undo* into the Command Box and press kbd:[Enter]:
 +
-image::UndoAndRedo/undocommand.png[, 550]
+image::UndoAndRedo/undocommand.png[, 500]
 
-. You will see a success message displays in the Result Box, and the transaction (Chicken Rice)
+. You will see a success message displays in the Result Box, and the transaction "Chicken Rice"
```
Comment (krusagiz) [inline docs/UserGuide.adoc]: It should be:
... see _that_ a success message ...

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -483,13 +475,15 @@ To `undo` the `delete` command:
...
 adds back to its original position in the list:
```
Comment (krusagiz) [inline docs/UserGuide.adoc]: You might want:
... _returns_ to its original position ...

File: docs/team/lightz96.adoc [hunk 7]
```diff
@@ -1,7 +1,120 @@
...
 :imagesDir: ../images
+:sectnums:
 :stylesDir: ../stylesheets
 :repoURL: https://github.com/AY1920S1-CS2103T-W12-2/main
 
-== Project: THRIFT
+== Introduction
+This portfolio aims to showcase the work that I have contributed to a team project called THRIFT. This project is an assignment under one of the core
+modules called CS2103T Software Engineering, offered by the School of Computing (SoC) at the National University of Singapore (NUS).
+I take this module during Year 2 as an undergraduate while pursuing my Bachelor of Computer in Computer Science at NUS.
```
Comment (krusagiz) [inline docs/team/lightz96.adoc]: It should be:
I _took_ this module...

File: docs/team/lightz96.adoc [hunk 8]
```diff
@@ -1,7 +1,120 @@
...
+include::../UserGuide.adoc[tag=undo]
+
+(End of extract)
+
+Please click on these links to view other sections that I have also contributed to the User Guide: https://ay1920s1-cs2103t-w12-2.github.io/main/UserGuide.html#RedoTag[Redoing the undone command], https://ay1920s1-cs2103t-w12-2.github.io/main/UserGuide.html#HelpTag[Displaying help page]
+
+[[DeveloperTag]]
+== Contribution to Developer Guide
+
+The following section demonstrates my documentation skills in providing technical details regarding the implementation of the undo and redo features.
```
Comment (krusagiz) [inline docs/team/lightz96.adoc]: You might want:
... technical details _such as_ the implementation ...

File: docs/team/lightz96.adoc [hunk 9]
```diff
@@ -1,7 +1,120 @@
...
+`**THRIFT**`:: Bold text with mark-up indicates a class/package name.
+
+== Summary of contributions
+This section summarises my contributions to THRIFT.
+
+* *Main feature*: I implemented the ability to undo and redo the previous command.
+** What it does: The undo command will undo the previous command while the redo command will redo the undone command.
+** Justification: It offers users the ability to perform undo and redo multiple times in the event
+that they are unable to make a decision. For example, users cannot decide how much budget to set for this month.
+** Highlights: The implementation is challenging because I need to provide different sets of implementation for each undoable command
```
Comment (krusagiz) [inline docs/team/lightz96.adoc]: How about:
... provide different _implementations_ ...

**Review by krusagiz — CHANGES_REQUESTED** [review]
LGTM after fixing grammar.

File: docs/DeveloperGuide.adoc (hunk 3 above)
Comment (lightz96) [reply docs/DeveloperGuide.adoc]: Fixed

File: docs/DeveloperGuide.adoc (hunk 4 above)
Comment (lightz96) [reply docs/DeveloperGuide.adoc]: Fixed

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (lightz96) [reply docs/DeveloperGuide.adoc]: Fixed

File: docs/UserGuide.adoc (hunk 5 above)
Comment (lightz96) [reply docs/UserGuide.adoc]: Fixed

File: docs/team/lightz96.adoc (hunk 7 above)
Comment (lightz96) [reply docs/team/lightz96.adoc]: Fixed

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (lightz96) [reply docs/DeveloperGuide.adoc]: Fixed

File: docs/team/lightz96.adoc (hunk 9 above)
Comment (lightz96) [reply docs/team/lightz96.adoc]: Fixed

File: docs/UserGuide.adoc (hunk 6 above)
Comment (lightz96) [reply docs/UserGuide.adoc]: Fixed

File: docs/team/lightz96.adoc (hunk 8 above)
Comment (lightz96) [reply docs/team/lightz96.adoc]: Fixed

**Review by krusagiz — APPROVED** [review]

**Pull request merged**